app/views/ufos/index.html.erb

```diff
@@ -10,14 +10,13 @@
 <div class="cards container mt-5">
     <% @ufos.each do |ufo| %>
       <div class="card" style="width: 18rem;">
-        <%= image_tag "ufo1.jpg", alt: "mojito", class: "card-img-top" %>
+        <%= image_tag "ufo1.jpg", alt: "ufo-image", class: "card-img-top" %>
         <%#= cl_image_tag photo_or_fallback(cocktail), style: "height: 180px; object-fit: cover;" %>
+          <p class="card-text card-absolute"><%= ufo.brand %></p>
         <div class="card-body">
-          <h5 class="card-title"><%= ufo.brand %></h5>
-          <p class="card-text"><%= ufo.speed %></p>
+          <p class="card-text"><%= ufo.description %></p>
+          <p class="card-text"><strong><%= ufo.price %></strong> Alien money</p>
```

Comment:
`<strong><%= ufo.price %></strong>` change to `<strong><%= ufo.price %>€</strong>` to have currency, we can add a way to change that dynamically later based on user location/country.``

Reply:
Didn't pull the branch to check it out. All good for me now.